torch/testing/_internal/common_distributed.py

```diff
@@ -953,3 +954,26 @@ def perThreadTearDown(self):
     @property
     def world_size(self) -> int:
         raise RuntimeError("world size not implemented")
+
+
```

Comment:
I see: So this `CheckPrecisionModule` does not check precision itself -- it is used to check the heterogeneous mixed precision case externally.

Reply:
Let me update the name to match that.